build/int.cloudbuild.yaml

```diff
@@ -22,6 +22,12 @@ steps:
   - 'TF_VAR_org_id=$_ORG_ID'
   - 'TF_VAR_folder_id=$_FOLDER_ID'
   - 'TF_VAR_billing_account=$_BILLING_ACCOUNT'
+
+- id: init-all
+  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  args: ['/bin/bash', '-c', 'cft test run all --stage init --verbose']
```

Comment:
question: Since both sections below have an `init` step, what is this `init-all` doing?

Reply:
I'm not sure, this was copied from https://github.com/GoogleCloudPlatform/terraform-google-secure-cicd/blob/main/build/int.cloudbuild.yaml#L31 as a presumed 'best practice' (note: only example I could find)

Reply:
I'm chatting with tf blueprint maintainers around best practices here, tbd